.rubocop.yml

```diff
@@ -22,5 +24,13 @@ Metrics/ClassLength:
 Metrics/MethodLength:
   Max: 20
 
+RSpec/DescribeClass:
+  Exclude:
+    - spec/tasks/**/*
+    - spec/system/**/*
+
+RSpec/MessageSpies:
+  EnforcedStyle: receive
```

Comment:
I actually like the alternative a bit better, but it's a lot of work. Just FYI, though:

Instead of something like this

```ruby
expect(:my_thing).to receive(:a_method)
:my_thing.a_method
```

You can do this:

```ruby
thing = spy(:my_thing)
thing.a_method
expect(thing).to have_received(:a_method)
```

The reason I like this is that it gets rid of the need to reverse the setup/expectation order when dealing with mocks. You still setup the test, perform the action, then assert the expectation — same order no matter what.